src/app/screens/Receive/index.tsx

```diff
@@ -100,6 +100,13 @@ function Receive() {
       });
   }
 
+  function receiveAnotherPayment() {
+    setPaid(false);
```

Comment:
formData should be defaulted; otherwise, we'll end up with the same amount and description on clicking "Create Invoice" the second time.